playwright/src/test/java/com/microsoft/playwright/TestPageNetworkSizes.java

```diff
@@ -53,13 +53,12 @@ void shouldSetBodySizeTo0IfThereWasNoBody() {
   }
 
   @Test
-  @Disabled("responseBodySize == 16")
   void shouldSetBodySizeHeadersSizeAndTransferSize() throws ExecutionException, InterruptedException {
     server.setRoute("/get", exchange -> {
       // In Firefox, |fetch| will be hanging until it receives |Content-Type| header
       // from server.
       exchange.getResponseHeaders().add("Content-Type", "text/plain; charset=utf-8");
-      exchange.sendResponseHeaders(200, 0);
+      exchange.sendResponseHeaders(200, 6);
```

Comment:
there seems unfortunately no better way to do this.

Reply:
"If the response length parameter is {@code zero}, then chunked transfer encoding is used..." - yeah, this is unfortunate